src/soilbiogeochem/SoilBiogeochemDecompMod.F90

```diff
@@ -221,25 +220,8 @@ subroutine SoilBiogeochemDecomp (bounds, num_soilc, filter_soilc,
             end do
          end do
       end do
-   else
```

Comment:
yes, this is is an example of the main purpose of the PR.  Also: noting that this is addressing an old post, since then the team has agreed to not splitting the PR.